certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -3929,474 +5023,1855 @@
+                                            },
+                                            "isPreferredBilling": {
+                                                "type": "boolean",
+                                                "description": "Addresses - Is Preferred Billing.",
+                                                "title": "Is Preferred Billing"
+                                            },
+                                            "isBadAddress": {
+                                                "type": "boolean",
+                                                "description": "Addresses - Is Bad Address.",
+                                                "title": "Is Bad Address"
+                                            }
+                                        }
+                                    },
+                                    "description": "Locations - Addresses.",
+                                    "title": "Addresses"
+                                }
+                            }
+                        },
+                        "description": "Event - Locations.",
+                        "title": "Locations"
+                    },
+                    "registrationDate": {
+                        "type": "string",
+                        "description": "Event - Registration Date.",
+                        "title": "Registration Date"
+                    },
+                    "customFields": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string",
+                                    "description": "Custom Fields - Name.",
+                                     "title": "Name"
+                                },
+                                "caption": {
+                                    "type": "string",
+                                    "description": "Custom Fields - Caption.",
+                                     "title": "Caption"
+                                },
+                                "value": {
+                                    "type": "string",
+                                    "description": "Custom Fields - Value.",
+                                     "title": "Value"
+                                }
+                            }
+                        },
+                        "description": "Event - Custom Fields.",
+                        "title": "Custom Fields"
+                    }
                 },
-                "secret": {
-                  "type": "string",
-                  "description": "Secret.",
-                  "title": "Secret",
-                  "x-ms-visibility": "internal",
-                  "default": "Impexium"
+                "description": "Event.",
+                "title": "Event"
```

Comment:
Please remove the title here as it's added to the name of the children properties.